webapp/channels/src/components/post/post_error_boundary.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import withErrorBoundary from 'components/with_error_boundary';
+
+export function withPostErrorBoundary<P>(component: React.ComponentType<P>) {
+    return withErrorBoundary<P>(component, {
+        renderFallback: ({clearError}) => {
+            return (
+                <div className='a11y__section post'>
+                    <FormattedMessage
+                        id='post.renderError.message'
+                        defaultMessage='An error occurred while rendering this post.'
+                        tagName='p'
+                    />
+                    <br/>
+                    <button
+                        className='btn btn-tertiary'
+                        onClick={clearError}
```

Comment:
`        aria-label='Retry rendering the post'
`

Reply:
Good call. I was hoping I could find a shorter way of saying that, but I can't come up with anything that's only a couple words and explains what's happening without more context.

I'll also change the actual text on the button to "Retry" to match the label since you usually want the visible text to be very similar for users who are at least partially sighted who might be using a11y tools